angular/core/components/group_page/group_theme/group_theme.haml

```diff
@@ -1,6 +1,6 @@
 .group-theme
-  .group-theme__cover{ng-style: "coverStyle()"}
-    .group-theme__upload-photo{ng-mouseover: "themeHoverIn()", ng-mouseleave: "themeHoverOut()", ng-if: "canUploadPhotos()"}
+  .group-theme__cover.group-theme__trigger-upload{ng-style: "coverStyle()", ng-mouseover: "themeHoverIn()", ng-mouseleave: "themeHoverOut()", ng-if: "canUploadPhotos()"}
```

Comment:
I don't really follow, but we can't be doing css manipulations in angular controllers unless we really need to. Let's close for now, as it's not that high a priority.